src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1020,3 +1020,9 @@ def update_acs(resource_group_name, container_service_name, new_agent_count):
     return client.container_service.create_or_update(resource_group_name,
                                                      container_service_name, instance)
 
+def list_container_services(resource_group_name=None):
+    ''' List Container Services. '''
+    ccf = _compute_client_factory()
```

Comment:
... Then in this method:
- Remove `ccf = _compute_client_factory()` and `ccf.container_services`
- The first argument will now be the container service client.
```
def list_container_services(client, resource_group_name=None):
   svc_list = client.list_by_resource_group(resource_group_name=resource_group_name) \
       if resource_group_name else ccf.container_services.list()
    return list(svc_list)
```